src/endpoints/create.rs

```diff
@@ -331,15 +346,28 @@ pub async fn create(
     };
 
     if encrypt_server {
-        Ok(HttpResponse::Found()
-            .append_header(("Location", format!("/auth/{}/success", slug)))
-            .finish())
+        Ok(Response::builder()
+            .status(302)
```

Comment:
You can use a constant (named FOUND) here, see:
https://docs.rs/http/1.3.1/http/status/struct.StatusCode.html

Should make things a little more readable.